packages/nextjs/components/scaffold-eth/Input/AddressInput.tsx

```diff
@@ -78,6 +97,18 @@ export const AddressInput = ({ value, name, placeholder, onChange, disabled }: C
             ) : null}
             <span className="text-accent px-2">{enteredEnsName ?? ensName}</span>
           </div>
+        ) : (
+          (isEnsAddressLoading || isEnsNameLoading) && (
+            <div className="flex bg-base-300 rounded-l-full items-center">
+              <div className="w-[35px]">
+                <div className="h-[35px] w-[35px] rounded-full bg-gray-500 animate-pulse"></div>
+              </div>
+              <div className="text-accent px-2 flex items-end space-x-1">
+                <span>Resolving</span>
```

Comment:
This is a bit too strong I think, Maybe we could:

- Leave the loading dots only
OR
- Use daisy skeleton: https://daisyui.com/components/skeleton/#skeleton---circle-with-content

The 2 option with the circle + bar will like great I think

Reply:
voting for loading dots only

Reply:
fixed at https://github.com/scaffold-eth/scaffold-eth-2/pull/738/commits/6efd33101fbdc0b34e1921e3e48e7ea9f61a4a2e